### PR TITLE
💄 Homepage - Landing hero remodel 

### PR DESCRIPTION
**Pull request by joshbermanssw**
As per #1808 we have cleaned up the landing screen on [tina.io](https://tina.io) as per the [figma design ](https://www.figma.com/design/Jyj0SszjfSBc6Minkl3g1l/Tina.io?node-id=399-5400&t=Jg6TgnnQLbyZ4aA9-0).

We have changed
- [ ] Removed one CTA as it is now in the nav bar 
- [ ] Increased size of left container marginally to include 2 CTAs on one row 
- [ ] Added floating play button that links to [Tina Youtube](https://youtube.com/tinacms)

Screenshot 2024-06-05 at 4 45 34 pm

**Figure: 💄Re-styled homepage**

### CODE REVIEW

File: components/blocks/Features.tsx [hunk 1]
```diff
@@ -31,25 +37,39 @@ export function FeatureBlock({ data, index }) {
...
-              // width="1120px"
-              // height="800px"
-              />
+              <>
+                <img
+                  src={data.media[0].image}
+                  alt={data.headline}
+                  className="w-full h-auto rounded-lg shadow-panel overflow-hidden bg-transparent"
+                />
+              </>
```
Comment (bradystroud) [inline components/blocks/Features.tsx:55]: any reason we aren't using [Next image](https://nextjs.org/docs/pages/api-reference/components/image)?

Maybe make a PBI to use it later

File: components/blocks/Features.tsx [hunk 2]
```diff
@@ -1,24 +1,30 @@
...
+import playImage from '../../public/img/playButton.png';
 
 export function FeatureBlock({ data, index }) {
-  const isReversed = index % 2 === 1
+  const isReversed = index % 2 === 1;
+
+const handlePlayClick = () => {
+  const url = "https://youtube.com/tinacms"
+  window.open(url, '_blank');
+}
```
Comment (bradystroud) [inline components/blocks/Features.tsx:15]: why not use an `<a>` instead of a button if it just opens a link

**Review by bradystroud — CHANGES_REQUESTED** [review]

File: components/blocks/Features.tsx (hunk 1 above)
Comment (joshbermanssw) [reply components/blocks/Features.tsx:55]: https://github.com/tinacms/tina.io/issues/1832 

✅

File: components/blocks/Features.tsx (hunk 2 above)
Comment (joshbermanssw) [reply components/blocks/Features.tsx]: fixed

**Comment by joshbermanssw** [comment]
test passed by @bettybondoc and @landonmaxwell

**Review by bradystroud — APPROVED** [review]
LGTM 
For the record, I dont like the floating play button 👎🏼
But it was approved by @bettybondoc and @landonmaxwell so I'm approving the code changes

**Comment by joshbermanssw** [comment]
> LGTM For the record, I dont like the floating play button 👎🏼 But it was approved by 2 designers so I'm approving it

In agreeance about the floating play button

**Comment by bettybondoc** [comment]
@bradystroud @joshberman1 
cc @jaydenalchin @landonmaxwell

> For the record, I dont like the floating play button 👎🏼

Is it just the colours, the icon used, or the idea of the floating play button altogether?

I don't have very strong opinion about it (at the very least, I feel like we will change the colours when we get the new GIF so that the button would pop out more) so happy to make cosmetic changes to it 😉

**Comment by bradystroud** [comment]
Hey @bettybondoc I've moved this discussion to Teams as the PR is merged

**Pull request merged**